app/src/main/java/org/oppia/app/profile/ProfileChooserViewModel.kt

```diff
@@ -65,4 +68,36 @@ class ProfileChooserViewModel @Inject constructor(
 
     return sortedProfileList
   }
+
+  fun addViewVisibility(binding: ProfileChooserAddViewBinding, boolean: Boolean) {
```

Comment:
Please name variables correctly.
` wasProfileEverCreated: Boolean`

Reply:
Also, you don't need binding parameter to update UI using viewModel. You can directly use an `Observable` boolean and control its value in this ViewModel which will then be responsible for controlling the UI. Check otehr viewmodels for correct implementation. Also, you will notice that none of the other viewmodels contain any `binding` in ViewModel, because that is not a correct approach.

Reply:
Okay, PTAL :)